release/models/vlan/openconfig-vlan-types.yang

```diff
@@ -63,7 +73,6 @@ module openconfig-vlan-types {
   }
 
   // OpenConfig specific extensions for module metadata.
-  oc-ext:regexp-posix;
```

Comment:
Since this is a module-level modifier could you also help convert `qinq-id` and `qinq-id-range` and add tests for them?